### ISSUE TITLE
Cannot find type definition file for request

### ISSUE THREAD

**Issue by prolink007** [njl07/rx-http-request#8]
Tried using the library and when compiling typescript i get the following error.
```

node_modules/@akanass/rx-http-request/lib/RxCookieJar.d.ts(1,1): error TS2688: Cannot find type definition file for 'request'.
node_modules/@akanass/rx-http-request/lib/RxHttpRequest.d.ts(1,1): error TS2688: Cannot find type definition file for 'request'.
```

It appears like the types for request are not being imported as a dependency.

**Comment by prolink007** [comment]
Fixed in #9

**Comment by njl07** [comment]
Thanks @prolink007 it's done
